pkg/controller/controller.go

```diff
@@ -41,8 +44,16 @@ import (
 	listers "github.com/kubernetes-incubator/service-catalog/pkg/client/listers_generated/servicecatalog/v1alpha1"
 )
 
-// NewController returns a new Open Service Broker catalog
-// controller.
+const (
+	// maxRetries is the number of times a resource add/update will be retried before it is dropped out of the queue.
+	// With the current rate-limiter in use (5ms*2^(maxRetries-1)) the following numbers represent the times
+	// a resource is going to be requeued:
+	//
+	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
+	maxRetries = 15
```

Comment:
Really cool! I'd like to have something built into the client or usable for tests as well.